powersimdata/design/transmission/upgrade.py

```diff
@@ -176,7 +176,7 @@ def get_branches_by_area(grid, area_names, method="either"):
 
     branch = grid.branch
     selected_branches = set()
-    grid_model = grid.get_grid_model()
+    grid_model = grid.grid_model
     for a in area_names:
         load_zone_names = area_to_loadzone(grid_model, a)
```

Comment:
Can we use `grid.grid_model` here and remove ` grid_model = grid.grid_model` since there is the only place where it is used